packages/typespec-vscode/snippets.json

```diff
@@ -0,0 +1,37 @@
+{
+  "Alias": {
+    "prefix": "alias",
+    "body": "alias ${1:AliasName} = ${0:Expression};",
```

Comment:
I don't know much about snippet but shouldn't those numbers be in order? Do they not map to which one gets filled first?

Reply:
The sequence starts from `1`, `2`...

`0` is a special index which is always the last one to reach. And when you press `Tab` to `0`, the iteration ends.

Reply:
I put `0` in all snippets, because those snippets have placeholders which may require free-style input, not just a single construct. For example, in the alias definition, the `Expression` could be a quite long input instead of just a model name.

Reply:
i see makes sense, thanks for explaining